### PR TITLE
Equip clothing wearing slowdown

### PR DESCRIPTION
**Pull request by Ghilker**
## About The Pull Request
Now there is a little delay before being able to wear an item of clothing. Different delays for different items.

## Why It's Good For The Game
Less clothing combat swapping strats (goes in conjunction with what people brought up in #58407

## Changelog
:cl:
balance: clothes will take some time before being worn
/:cl:

### CODE REVIEW

**Comment by SmArtKar** [comment]
That's just stright up cancerous solution

**Comment by SuperNovaa41** [comment]
I can't tell if this is a genuine change you want or you're just doing it because Oranges said he would approve it

**Comment by Ghilker** [comment]
> I can't tell if this is a genuine change you want or you're just doing it because Oranges said he would approve it

y'all know i CRAVE for Oranges approval

**Comment by SuperNovaa41** [comment]
> 
> 
> > I can't tell if this is a genuine change you want or you're just doing it because Oranges said he would approve it
> 
> y'all know i CRAVE for Oranges approval

Your last two PRs have been exactly that

File: code/modules/clothing/spacesuits/_spacesuits.dm [hunk 1]
```diff
@@ -46,6 +46,7 @@
 	equip_delay_other = 80
 	resistance_flags = NONE
 	actions_types = list(/datum/action/item_action/toggle_spacesuit)
+	equip_time = 1.5 SECONDS
```
Comment (necromanceranne) [inline code/modules/clothing/spacesuits/_spacesuits.dm:49]: I know we just had a conversation about equipping suits in space being a problem while moving, but actually I think space suits taking even longer than 1.5 seconds is actually probably a better representation of their bulk and cumbersome nature, and really makes the suits that don't have slowdown significantly more desirable and premium for their utility.

I don't think hardsuits should be things you just hotswap into easily, as an example.

Comment (Ghilker) [reply code/modules/clothing/spacesuits/_spacesuits.dm:49]: increased to 4.5 seconds

**Comment by moocowswag** [comment]
This would also lower the skill ceiling when it comes to how quickly an antag can get into a new disguise and every second tends to matter.

Im not saying its a good thing or a bad thing I'm just stating other effects of this pr.

**Comment by Mothblocks** [comment]
> Your last two PRs have been exactly that

tbf the first one was a request from me and rohesie

File: code/modules/clothing/spacesuits/_spacesuits.dm (hunk 1 above)
Comment (Mothblocks) [reply code/modules/clothing/spacesuits/_spacesuits.dm]: this kills tactical space battery swapping

**Comment by zxaber** [comment]
Indirect buff to the chameleon kit.

File: code/modules/clothing/masks/_masks.dm [hunk 2]
```diff
@@ -5,6 +5,7 @@
 	slot_flags = ITEM_SLOT_MASK
 	strip_delay = 40
 	equip_delay_other = 40
+	equip_delay_self = 0.6 SECONDS
```
Comment (TemporalOroboros) [inline code/modules/clothing/masks/_masks.dm:8]: This nerfs internals. Nitrous flood memes say hi.

Comment (Mothblocks) [reply code/modules/clothing/masks/_masks.dm:8]: of all people he's not going to care

File: code/modules/clothing/spacesuits/_spacesuits.dm (hunk 1 above)
Comment (necromanceranne) [reply code/modules/clothing/spacesuits/_spacesuits.dm]: the less we talk about spacesuit batteries the better.

File: code/modules/clothing/masks/_masks.dm (hunk 2 above)
Comment (Ghilker) [reply code/modules/clothing/masks/_masks.dm:8]: is less than a second

Comment (ArcaneDefence) [reply code/modules/clothing/masks/_masks.dm:8]: > 
> 
> of all people he's not going to care

HE CAN'T KEEP GETTING AWAY WITH IT!!

Comment (Ghilker) [reply code/modules/clothing/masks/_masks.dm:8]: sgQPw7n-Imgur

**Comment by Ghilker** [comment]
https://user-images.githubusercontent.com/42839747/114762509-22bb7b00-9d62-11eb-832c-c8b3516ffb27.mp4

**Comment by Ghilker** [comment]
emergency masks (the ones that comes inside the emergency boxes) have a instant wear

**Comment by MrPerson** [comment]
> That's just stright up cancerous solution

Normally I would agree actually, but as long as times are kept fairly short (like less than 1.5 seconds) for everything that isn't a space suit, armor, or useful in combat, we should generally be fine.

File: code/modules/clothing/gloves/_gloves.dm [hunk 3]
```diff
@@ -11,6 +11,7 @@
 	var/transfer_prints = FALSE
 	strip_delay = 20
 	equip_delay_other = 40
+	equip_delay_self = 0.6 SECONDS
```
Comment (ATH1909) [inline code/modules/clothing/gloves/_gloves.dm:14]: oh no, a 0.6 second delay. this will definitely stop me from combat-swapping my gloves

**Comment by ATH1909** [comment]
> > That's just stright up cancerous solution
> 
> Normally I would agree actually, but as long as times are kept fairly short (like less than 1.5 seconds) for everything that isn't a space suit, armor, or useful in combat, we should generally be fine.

I mean, the times add up when you're reequipping all of your gear after being, say, stripped.

**Comment by Mothblocks** [comment]
a round is an hour 30 minutes the 15 seconds tops you lose is gonna be ok in the long run

**Review by Ghilker — COMMENTED** [review]
no idea why this review to comment was here (ignore this one)

**Comment by Mothblocks** [comment]
that being said i don't really like this, I don't think it does anything about "combat swapping"--for insulated gloves, you'd just take them off, small delay, and then fight normally.

i think this just adds annoyance, and that's it.

**Review by Mothblocks — CHANGES_REQUESTED** [review]
guess i should press this button to say that huh

**Comment by Ghilker** [comment]
Nevermind

**Comment by interestingusernam3** [comment]
engineer gaming

**Pull request closed without merging**